Libplanet/Tx/Transaction.cs

```diff
@@ -601,10 +598,8 @@ internal RawTransaction ToRawTransaction(bool includeSign)
 
         private static T ToAction(IDictionary<string, object> arg)
         {
-            var typeStr = (string)arg["type_id"];
-            var action = (T)Activator.CreateInstance(Types[typeStr]);
-            var values = (IDictionary<string, object>)arg["values"];
-            action.LoadPlainValue(values.ToImmutableDictionary());
+            var action = new T();
+            action.LoadPlainValue(arg.ToImmutableDictionary());
```

Comment:
Can we deserialize the actions properly without concrete action type?

Reply:
Did you mean, what if `T` is not a concrete type?

Reply:
Right, what if we use like `DumbAction`?

Reply:
That's fine because `DumbAction` is a concrete class.  According to the [official docs][1], `new()` constraint is satisfied by only concrete class, neither `abstract class` nor `interface`:

> To use the new constraint, the type cannot be abstract.

That means, from now on we only accept a concrete class as an action type — even `PolymorphicAction<T>` in itself is concrete.  This is a point of getting rid of built-in subtype dispatcher.  Note that `T` of `PolymorphicAction<T>` does not have `new()` constraint on the other hand.  It can take an `abstract class` or an `interface`.

[1]: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/new-constraint